packages/shared/prisma/migrations/20240423124411_add_prompt_version_labels/migration.sql

```diff
@@ -0,0 +1,16 @@
+BEGIN;
+
+-- Step 1: Alter the 'prompts' table to add the 'labels' column
+ALTER TABLE prompts
+ADD COLUMN labels TEXT[] DEFAULT ARRAY[]::TEXT[];
+
+-- Step 2: Update the 'labels' column to include 'production' for active prompts
+UPDATE prompts
```

Comment:
I checked the database. With current data load, this statement should not time out.